src/ptxtransfer.cpp

```diff
@@ -257,7 +257,7 @@ class Mesh
         }
 
         verts.resize(vertcount/3);
-        for (uint i=0; i<verts.size(); i++) verts[i] = Vec3f(vp[i*3],vp[i*3+1],vp[i*3+2]);
+        for (uint32_t i=0; i<verts.size(); i++) verts[i] = Vec3f(vp[i*3],vp[i*3+1],vp[i*3+2]);
```

Comment:
The loop index caught my attention here. If we're going to touch these lines then we might prefer `size_t` over of `uint32_t` because we're comparing the value against `verts.size()` and friends.

I understand that the use of `uint` is pre-existing. There was probably a reason for it a long time ago.